src/lib.rs

```diff
@@ -279,8 +277,6 @@ macro_rules! iproduct {
 /// Prefer this macro `izip!()` over [`multizip`] for the performance benefits
 /// of using the standard library `.zip()`.
 ///
-/// [`multizip`]: fn.multizip.html
-///
```

Comment:
Could you explain why this commit removes some lines?

Reply:
In markdown if you write `[foo]` without defining `[foo]: bar` or writing `[foo](bar)`, the link will go directly to `foo`. That means that if the text of the link happens to already be valid according to intra-doc link rules, there is no need to provide a custom link target (this line would end up reading ``[`multizip`]: multizip`` and rustdoc knows enough to strip the backticks).